src/EnergyPlus/DataDaylighting.hh

```diff
@@ -63,27 +62,31 @@
 
 namespace EnergyPlus {
 
-namespace DataDaylighting {
+namespace Dayltg {
 
     // Two kinds of reference points: used directly in daylighting, used to show illuminance map of zone
     constexpr int MaxMapRefPoints(2500); // Maximum number of Illuminance Map Ref Points
 
-    enum class SkyType : int
+    enum class SkyType 
     {
         Invalid = -1,
-        Clear = 1,
+        Clear,
         ClearTurbid,
         Intermediate,
         Overcast,
         Num
     };
 
+    struct Illums {
+        std::array<Real64, (int)SkyType::Num> sky = {0.0, 0.0, 0.0, 0.0};
+    };
+        
     enum class ExtWinType
     {
         Invalid = -1,
-        NotInOrAdjZoneExtWin, // Exterior window is not in a Daylighting:Detailed zone or in an adjacent zone with a shared interior window
-        InZoneExtWin,         // Exterior window is in a Daylighting:Detailed zone
-        AdjZoneExtWin,        // Exterior window is in a zone adjacent to a Daylighting:Detailed zone with which it shares an interior window
+        NotInOrAdjZone, // Exterior window is not in a Daylighting:Detailed zone or in an adjacent zone with a shared interior window
```

Comment:
`ExtWin` is already in the type name, no need to have it in the type values also.